### DISCUSSION TITLE
I can't setup Dify alognside EasyPanel on DigitalOcean VPS

### DISCUSSION THREAD

**Discussion by klev-sm** [langgenius/dify#5762] [General]
I'm trying to self hosting my web app on a DigitalOcean droplet that already has EasyPanel installed, I don't know why but I can access other services like Weaviate and the api itself located on 5001 (that at least returns 404 to me), but when I try to go to the web URL, it does not works, I think that is related to some port conflicts, but I can't figure it out. I think that the only different thing that I change was the web port (previously 3000), now is 3001 because that EasyPanel uses it

This is my  `docker ps`:

![docker ps](https://github.com/langgenius/dify/assets/65031513/9363c7d8-64ed-43e9-bbcd-52ae50f23b32)

**Comment by taowang1993** [comment]
easypanel doesn't support complex docker compose

**Comment by bhctest123** [comment]
did this ever get resolve @taowang1993 or @klev-sm ?
